### DISCUSSION TITLE
Battery listed correctly on Device List but not showing up on pages nor VRM

### DISCUSSION THREAD

**Discussion by mhin76** [mr-manuel/venus-os_dbus-mqtt-battery#23] [Q&A] [answered]
Hi all.
I was able to implement successfully this great driver and it shows up correctly on device list:
![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/49210075/b4c08635-5c43-495d-be0f-cd4d03a32a1c)

however not on pages:
![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/49210075/63dfe725-90ec-46d6-8304-ff0ce7c4188a)

I have installed an emulated MultiPlus-II (not sure if that is in conflict) and added the battery service on the MultiPlus emulation service. That works (MultiPlus recognizes the battery %):
![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/49210075/8991f48d-7831-493d-b7b5-8d48ab97900e)

I am using the latest Venus os version v3.33 on Raspberry Pi 3. Any idea what I am doing wrongly?

**Comment by mr-manuel** [comment, marked as answer]
Probably your battery monitor is wrong. Check in the Remote Console/GUI -> Settings -> System setup -> Battery monitor.

**Reply by mhin76** [reply]
Thanks Manuel!! I knew I was missing somewhere a setting. Great, all looks fine. Awesome driver! thanks a lot!
![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/49210075/889a5780-b291-43e7-99cd-4c43a6d57d28)
